Cake.Recipe/Content/addins.cake

```diff
@@ -8,19 +8,19 @@
 #addin nuget:?package=Cake.Git&version=0.19.0
 #addin nuget:?package=Cake.Gitter&version=0.9.0
 #addin nuget:?package=Cake.Graph&version=0.6.0
-#addin nuget:?package=Cake.Incubator&version=2.0.2
+#addin nuget:?package=Cake.Incubator&version=3.0.0
 #addin nuget:?package=Cake.Kudu&version=0.8.0
 #addin nuget:?package=Cake.MicrosoftTeams&version=0.7.0
 #addin nuget:?package=Cake.ReSharperReports&version=0.10.0
 #addin nuget:?package=Cake.Slack&version=0.12.0
 #addin nuget:?package=Cake.Transifex&version=0.7.0
 #addin nuget:?package=Cake.Twitter&version=0.8.0
-#addin nuget:?package=Cake.Wyam&version=1.4.1
-#addin nuget:?package=Cake.Issues&version=0.4.0
-#addin nuget:?package=Cake.Issues.MsBuild&version=0.4.0
-#addin nuget:?package=Cake.Issues.InspectCode&version=0.4.0
-#addin nuget:?package=Cake.Issues.Reporting&version=0.4.0
-#addin nuget:?package=Cake.Issues.Reporting.Generic&version=0.4.0
+#addin nuget:?package=Cake.Wyam&version=1.5.1
+#addin nuget:?package=Cake.Issues&version=0.6.2
+#addin nuget:?package=Cake.Issues.MsBuild&version=0.6.1-beta0001
```

Comment:
@Jericho I assume this came through the Addin.Discoverer? IMHO it shouldn't update to pre-release versions.

Reply:
Created https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/32 for this